boards/st/stm32n6570_dk/stm32n6570_dk_common.dtsi

```diff
@@ -17,11 +17,14 @@
 	chosen {
 		zephyr,console = &usart1;
 		zephyr,shell-uart = &usart1;
-		zephyr,sram = &axisram2;
+		zephyr,sram = &axisram1;
```

Comment:
This should be done differently as both images are impacted.
Ideally a 3rd image should be created so we have 3 possible variants
/chainloaded: XIP or RAMLOAD with `zephyr,sram = &axisram1;`
/sb: RAMLOAD by boot ROM with `zephyr,sram = &axisram2;` loaded directly on RAM via serial boot
/fsbl: RAMLOAD by boot ROM with `zephyr,sram = &axisram2;` loaded on external flash

Issue is that when --sysbuild is used in theory /fsbl should be used for mcuboot and /chalinloaded should be used by application, but this doesn't see possible.